src/main/java/org/opentripplanner/model/FlexStopLocation.java

```diff
@@ -62,4 +86,23 @@ public String getFirstZoneAsString() {
   public void setZoneId(String zoneId) {
     this.zoneId = zoneId;
   }
+
+  @Override
+  public boolean isPartOfStation() {
+    return false;
+  }
+
+  @Override
+  public StopTransferPriority getPriority() {
+    return null;
```

Comment:
This should probably return `StopTransferPriority.ALLOWED`

Same goes for stop group, or maybe in that case it would be delegated to the individual stops?

Reply:
If we delegate it to the invidual stops, which value should we take if they don't have the same one?